cms/envs/common.py

```diff
@@ -465,6 +467,8 @@
 USE_I18N = True
 USE_L10N = True
 
+STATICI18N_ROOT = PROJECT_ROOT / "static"
```

Comment:
How will this work in production system?  We usually put the static folder in a different place than the default dev location.  I didn't see any changes in aws.py for dealing with updating this.  Also, is `'PROJECT_ROOT' / "static"` something that already exists or is this a completely new folder for i18n files?